MON-20534-refresh-snmp-protocol-topic (#2454)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md

---
id: applications-protocol-snmp
title: Generic SNMP
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **Generic SNMP** brings 2 host templates:

* **App-Protocol-SNMP-custom**
* **App-Protocol-SNMP-Only-custom**

<Tabs groupId="sync">
<TabItem value="App-Protocol-SNMP-custom" label="App-Protocol-SNMP-custom">

This host template does not bring any service template.

</TabItem>
<TabItem value="App-Protocol-SNMP-Only-custom" label="App-Protocol-SNMP-Only-custom">

> This host template does not come linked to any service template but it's host check command is based on a SNMP Uptime check, thus valid even when ICMP protocol is blocked.

Reviewer: ```suggestion
This host template does not bring any service template but its host check command is based on an SNMP uptime check, thus valid even when the ICMP protocol is blocked.
```

---
id: applications-protocol-snmp
title: Generic SNMP
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ## Pack assets

Reviewer: same comments as for the FR version

</TabItem>
</Tabs>

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics linked to each service.

<Tabs groupId="sync">
<TabItem value="Numeric-Value" label="Numeric-Value">

> Coming soon

Reviewer: coming soon

</TabItem>
<TabItem value="String-Value" label="String-Value">

> Coming soon

Reviewer: coming soon

| WARNING      | Warning threshold                                                                                   |                   |             |
| CRITICAL     | Critical threshold                                                                                  |                   |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector

> is able to monitor a server using a command like this one (replace the sample values by yours):

Reviewer: ```suggestion
is able to monitor a resource using a command like this one (replace the sample values by yours):
```
